internal/anchor/validator.go

```diff
@@ -3,6 +3,7 @@ package anchor
 import (
 	"context"
 	"fmt"
+	"k8s.io/apimachinery/pkg/util/validation"
```

Comment:
Can you please move this to the next "paragraph" (e.g. line 8 or lower)? The first "paragraph" is for golang standard libraries, the second is for other external dependencies, and the third is for HNC-internal packages.

Reply:
done, moved that. I should have noticed that. Apologies that's my bad.